libobs/obs-source.c

```diff
@@ -2417,8 +2417,7 @@ void obs_source_output_video(obs_source_t *source,
 		return;
 	}
 
-	struct obs_source_frame *output = !!frame ?
-		cache_video(source, frame) : NULL;
+	struct obs_source_frame *output = cache_video(source, frame);
```

Comment:
Was removing the check intentional?

Reply:
yes. value of frame is get checked few lines above so checking it again is misleading.

Reply:
got it